src/client/pages/Landing/components/Card.tsx

```diff
@@ -1,102 +1,138 @@
-import { css } from '@emotion/core'
 import styled from '@emotion/styled'
 import { colorsV3 } from '@hedviginsurance/brand'
 import React from 'react'
-import { useMediaQuery } from 'react-responsive'
 import { Link } from 'react-router-dom'
-import { ForwardArrow } from 'components/icons/ForwardArrow'
+import { Arrow } from 'components/icons/Arrow'
+import { HedvigSymbol } from 'components/icons/HedvigSymbol'
 
-const CardComponent = styled.div<{ banner?: boolean }>`
+const CardComponent = styled(Link)`
   position: relative;
   display: block;
   width: 100%;
-  margin-bottom: 1rem;
-  padding: 2.5rem 3rem 1rem 1rem;
-  padding-top: ${(props) => (props.banner ? '4rem' : '2.5rem')};
+  margin-bottom: 0.5rem;
+  padding: 1rem;
+  color: ${colorsV3.gray900};
   background: ${colorsV3.gray100};
   border-radius: 0.5rem;
+  box-shadow: 0px 2px 4px 0 rgba(0, 0, 0, 0.1);
   transition: all 0.35s;
   text-decoration: none;
 
   @media (min-width: 600px) {
-    padding: 2.5rem 2rem 1.5rem 1.5rem;
+    padding-left: 2rem;
+    padding-right: 2rem;
+  }
+
+  @media (min-height: 620px) {
+    margin-bottom: 1rem;
   }
 
   @media (min-width: 850px) {
     display: flex;
     flex-direction: column;
-    align-items: center;
     justify-content: space-between;
-    max-width: 29rem;
-    margin-top: ${(props) => (props.banner ? '-2rem' : 0)};
-    margin-right: 0.5rem;
-    margin-left: 0.5rem;
-    padding: 2rem;
-    padding-top: ${(props) => (props.banner ? '4.5rem' : '2.5rem')};
-    text-align: center;
+    max-width: 28rem;
+    margin: 0 0.5rem 1rem;
+    padding: 1.5rem;
   }
 
   @media (min-width: 1020px) {
-    display: flex;
-    align-items: center;
     justify-content: space-between;
-    margin-right: 1rem;
-    margin-left: 1rem;
-    padding: 2.5rem;
-    padding-top: ${(props) => (props.banner ? '5.5rem' : '3.5rem')};
+    margin-right: 0.75rem;
+    margin-left: 0.75rem;
+    padding: 2rem;
+    border-radius: 0.75rem;
   }
 
   :hover {
     transform: translateY(-6px);
+    color: ${colorsV3.gray900};
```

Comment:
Is this to reset some other style we have for the `Link` component or something?

Reply:
Yes, it's to make sure the svg get's the correct color on hover since it inherits with `currentColor` :)